### PR TITLE
Update codyCommit

### PR DESCRIPTION
**Pull request by mkondratek**
This PR is important b/c it brings a crucial fix to the telemetry 

## Test plan 
- test everything 🤡

### CODE REVIEW

File: src/main/kotlin/com/sourcegraph/cody/agent/ExtensionConfiguration.kt [hunk 1]
```diff
@@ -9,6 +9,8 @@ data class ExtensionConfiguration(
     val autocompleteAdvancedProvider: String? = null,
     val autocompleteAdvancedServerEndpoint: String? = null,
     val autocompleteAdvancedAccessToken: String? = null,
+    val customConfiguration: Map<String, Any> =
+        mapOf("cody.useContext" to "embeddings", "cody.experimental.symfContext" to false),
```
Comment (mkondratek) [inline src/main/kotlin/com/sourcegraph/cody/agent/ExtensionConfiguration.kt]: @olafurpg , does it make sense?

Comment (olafurpg) [reply src/main/kotlin/com/sourcegraph/cody/agent/ExtensionConfiguration.kt]: This looks right 👍🏻

File: src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt [hunk 2]
```diff
@@ -114,6 +111,9 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
     addWelcomeMessage()
     ApplicationManager.getApplication().executeOnPooledThread { refreshSubscriptionTab() }
     loadNewChatId()
+
+    // Initiate filling recipes panel in the background
+    refreshRecipes()
```
Comment (mkondratek) [inline src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt:118]: moved from above - probably no actual impact but looks better at the end. I will provide some cleanup to this class in a separate PR

File: src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt [hunk 3]
```diff
@@ -215,6 +215,7 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
   }
 
   private fun loadCommands() {
+    tryRestartingAgentIfNotRunning(project)
```
Comment (mkondratek) [inline src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt:225]: I tried `:runIde` a few times - with a rate-limited account I saw the issue with commands not loading - this line seems to be a fix

File: src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt [hunk 4]
```diff
@@ -91,6 +88,10 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
       stopGeneratingButton.isVisible = false
       sendButton.isEnabled = promptPanel.textArea.text.isNotBlank()
       ensureBlinkingCursorIsNotDisplayed()
+      recipesPanel.components.filterIsInstance<JButton>().forEach {
+        it.isEnabled = true
+        it.toolTipText = null
+      }
```
Comment (mkondratek) [inline src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt:94]: unrel fix

File: src/main/kotlin/com/sourcegraph/cody/agent/ExtensionConfiguration.kt (hunk 1 above)
Comment (mkondratek) [reply src/main/kotlin/com/sourcegraph/cody/agent/ExtensionConfiguration.kt]: we decided to just cherry pick the telemetry fix - these adjustments are redundant

**Review by pkukielka — APPROVED** [review]
LGTM

**Review by exigow — CHANGES_REQUESTED** [review]
Chat input seems to be infinitely locked on "Connecting to agent" on private repositories. On public repos like sourcegraph/jetbrains/cody/etc this problem doesn't exists (chat connects to the agent and after a moment the text input becomes usable).

Selection_033

I've double-checked this and it appears that the current main branch doesn't have this issue, so the regression must have been introduced with this PR (which seems very strange since the changes are cherry-picked and are solely related to telemetry).

Does anyone have similar experience?

EDIT/NOTE: Sometimes it's working if we launch a private repo first. The problem is fully reproducible if we open another private repo within the same "session".

**Review by exigow — APPROVED** [review]
Works great. The last fix nailed it. :+1:

**Review by Sa1to — APPROVED** [review]

**Pull request merged**